.github/workflows/ismrmrd_python_conda.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request:
+    branches:    
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build-conda-packages:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Probably more of a theoretical issue, but we can reference github actions by commit hash instead of version (which is mutable), which can help protect against malicious updates.